src/packages/auth/hoc.tsx

```diff
@@ -13,8 +14,12 @@ const auth = (WrappedComponent: any) => {
 		authType: string;
 		roles: string[] | null;
 	}) => {
-		if (authType === OPEN_ID_CONNECT_AUTH)
-			return <LoginOidcComponent WrappedComponent={WrappedComponent} />;
+		if (authType === OPEN_ID_CONNECT_AUTH) {
+			const { isUserLoggedIn } = useOidc();
+			if (!isUserLoggedIn)
+				return <LoginWrapper WrappedComponent={WrappedComponent} />;
```

Comment:
I think we do not need to pass the WrappedComponent in this case, because It won't be displayed. What do you think ?